java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -1291,4 +1291,20 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_title(JNIEnv *env, jobj
   CATCH_STD(env, 0);
 }
 
+JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_nansToNulls(JNIEnv *env, jobject j_object, jlong handle) {
+
+  JNI_NULL_CHECK(env, handle, "native view handle is null", 0)
+
+  try {
+    cudf::jni::auto_set_device(env);
+    cudf::column_view * view = reinterpret_cast<cudf::column_view *>(handle);
+    auto pair = cudf::experimental::nans_to_nulls(*view);
+    cudf::bitmask_type const* data = reinterpret_cast<cudf::bitmask_type const*>(pair.first->data());
```

Comment:
You did what @jlowe asked, but the code is kind of hard to follow.  A comment explaining what you are doing and why would be helpful.

Reply:
I also recommend calling this local "mask" or "validity" or something to indicate what it is.  Just calling it `data` makes it easy to confuse with the data vector.